docs/components/modeler/web-modeler/advanced-modeling/camunda-docs-ai.md

```diff
@@ -0,0 +1,19 @@
+---
+id: camunda-docs-ai
+title: Camunda Docs AI
```

Comment:
Since we aren't pulling _only_ from the docs for responses, I'm uncertain about the naming here, particularly so this isn't mixed up with Kapa in the actual docs.

Will this instead be referenced as the Camunda Wizard, as referenced in the related issue? Or, are we looking for additional naming suggestions?

Reply:
If I saw "Camunda Docs AI" in the sidebar, I might think it is a page about AI in the Camunda documentation, not a source for assistance in the actual product UI.

Reply:
CC @akeller, since you're the Kapa go-to in the docs.

Reply:
cc: @HanselIdes

Reply:
I'm OK with the name because it's the same feature, just brought into a different context.

@bastiankoerber do you have any guidance for AI-powered feature naming?

Reply:
For now, I will go with the current name based on the suggestion from the PM.